templates/valid_example.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
- the file should live in `tests/assets/`
- `valid_markup_example.html` would be more good name for the file